docs/signals/configuration/stream-calculations/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Calculate in real-time"
+sidebar_position: 49
+---
+
+To calculate attributes in real time, apply a default (without specified `batch_source`) [view or service](/docs/signals/configuration/views-services/index.md) configuration to Signals.
+
+Apply the configuration using the `apply` method:
+
+```python
+from snowplow_signals import Signals
+
+# Connect to Signals
+# See the main Configuration section for more on this
+sp_signals = Signals(
+        {{ config }}
+    )
+
+# Apply views
+sp_signals.apply([
+        my_view,
+        my_other_view
+    ])
+
+# Apply services
+sp_signals.apply([
+        my_service,
+        my_other_service
+    ])
+```
+
+Once applied, Signals will calculate the attributes within these views or services from the Snowplow event stream.
+
+If you only want to publish the attribute definitions, and don't want to calculate the attribute values now, use the `online=False` [view configuration option](/docs/signals/configuration/views-services/index.md).
+
+## Stream volume limit
```

Comment:
This is only partly true – the differences are more nuanced. I have added a commit in the other PR with an overview table comparing the limitations of stream vs batch processing (sorry I made a conflict for your PR): https://github.com/snowplow/documentation/pull/1197/commits/05cabb6e50ee8a3ef826cf8c7a22feb32e732c39

I am not sure if this is the best place to put the discussion of limitations of stream vs batch, I think I would prefer if this and the next page were grouped under a common top level page giving an introduction to the two types of processing and discussing their limitations as well. I think we are missing some overview page here before jumping into the options.